### DISCUSSION TITLE
Speed Estimation Example in Windows 11.

### DISCUSSION THREAD

**Discussion by Atomix2402** [roboflow/supervision#972] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Hi Roboflow, I'm a huge fan of your work !! As I'm a beginner to Computer vision and Object detection I'm astounded by the supervision library I'm always thankful for that. 
Recently I was working on the Traffic Analysis project, but instead of MacBook I was working on Windows 11. I don't know key changes I have to make to get the correct output in my System. But I was trying my best to do with whatever I can, but in windows 11 I'm getting error when i was trying to capture video from the CV2 library. I don't understand what to do. 
I don't know how to operate with Arg parser in Windows 11 when I researched about it I found out I have to install various Bash language scripts or something but i wanted to know if that could be avoided. So i want a windows 11 tutorial if possible or else help me with that. I have also commented in youtube under Piotr Skalski's (big fan)  video. Please help me.
![image](https://github.com/roboflow/supervision/assets/112922417/2b074061-3aca-4fc8-9758-6cde2d7cf3a4)
![image](https://github.com/roboflow/supervision/assets/112922417/a6a7e22c-0e66-40bf-ab4d-9b5edd358c4b)
![image](https://github.com/roboflow/supervision/assets/112922417/20a7c04c-4de9-48b2-acc2-2e6ac7e7897e)

**Comment by onuralpszr** [comment]
Converting Question to Discussion Q&A

**Reply by Atomix2402** [reply]
Can you help me with the above problem ?
